prem/VennDiagram.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+    <html>
+    <head>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>
+            VennDiagram
+        </title>
+        <style>
+            .main-section {
+            position: absolute;
+            width: 30%;
+            height: 30%;
+            margin: auto;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+        }
+        .green-circle,.orange-circle,.pink-circle{
+            width: 50%;
+            height:75%;
+            border-radius: 50%;
+            position: absolute;
+        }
+        .green-circle{
+            background-color:#00fb00;
+            display: inline-block;
+            margin-left:5%;
+
+        }
+        .orange-circle{
+            background-color: #f1a300;
+            display: block;

```

Comment:
Why you have added "display:block" ?

Reply:
Basically div is a block element, so this line is not necessary. In order to make myself clear in the idea of creating the Venn diagram on my first version i used to this statement to make the diagram possible.
So, i translated my first version into the center element so i inherited the line.
I will change it and merge it.